src/main/java/com/zebrunner/agent/junit5/JUnit5Adapter.java

```diff
@@ -45,14 +45,18 @@ public void registerRunFinish(TestPlan testPlan) {
 
     public void registerTestStart(TestIdentifier testIdentifier) {
         if (testIdentifier.isTest()) {
-            OffsetDateTime startedAt = OffsetDateTime.now();
+
             String uuid = testIdentifier.getUniqueId();
```

Comment:
It would be ideal if you renamed the `uuid` property to something like `correlationData` to follow the new notation